GliaWidgets/Interactor/Interactor.swift

```diff
@@ -160,33 +161,6 @@ extension Interactor {
         }
     }
 
-    func request(
```

Comment:
This method is no longer needed?

Reply:
It was never used. I think it might become useful again in the future when visitor can request media upgrade as well, but then it can be implemented again. Redundant for now.